tool/src/main/java/org/wildfly/security/tool/MaskCommand.java

```diff
@@ -71,6 +72,10 @@
 
     @Override
     public void execute(String[] args) throws Exception {
+        if (new SecureRandom().getProvider().getName().toLowerCase().contains("fips")) {
```

Comment:
I decided to make FIPS check weaker so there won't be warning everytime for users who are not using FIPS (see explanation in previous comment). Also, after Java 11 the `com.sun.net.ssl.internal.ssl.Provider` is not available.